doc: Fix various documention issues (#1566)
aristanetworks/anta: docs/cli/overview.md

```

!!! note
    All environment variables may not be needed for every command.

>     Refer to `<command> --help` for the comprehensive environment variable names.

Reviewer: ```suggestion
    Refer to `<command> --help` for a comprehensive list of environment variables.
```
